FRPs/proposal-2-liquidations.md

```diff
@@ -0,0 +1,50 @@
+---
+id: 2
+title: MEV and Liquidations
+team: Kaihua Qin, Liyi Zhou, Arthur Gervais
+created: 2020-12-10
+paper: 1
+---
+
+# Summary
+With over 6B USD total value locked for the top 3 lending protocols alone, this use case has grown into a substantial DeFi market. Lending entails the risks of a diminishing collateral value, which may render debt unhealthy. Unhealthy debt is typically resolved through liquidations, a process of selling collateral at a discount to liquidators.
+
+One dominant liquidation design is the fixed spread liquidation mechanism, used by Aave, Compound and dYdX. Interestingly, the fraction of the collateral that a liquidator can purchase at a discount amounts to 50% for Aave and Compound, and even 100% for dYdX (commonly referred to as close factor). It therefore appears that the fixed spread liquidation mechanisms are benefitting the liquidators more than the borrowers.
+
+Drawing the link to MEV, each liquidation event may qualify as Miner Extractable Value (MEV). For example, in a recent price oracle manipulation attack, a liquidator performed an oracle price post and liquidated millions of USD within one transaction, yielding a profit of nearly 4M USD. 
+
+The goal of this FRP is to offer a systematization of existing liquidation mechanisms and consistent measurements of the past and current DeFi liquidation state. We aim to investigate alternative liquidation designs in an effort to mitigate MEV from liquidation events.
+
+## Background and Problem Statement
+
+The goal of this FRP is to study the past and current status of blockchain liquidation. Specifically, we would like to answer the following questions:
+- What is the monthly accumulated liquidation profit for Aave, Compound, dYdX, MakerDAO?
+- What gas price distributions are being paid for the liquidation events?
+- What are the different liquidation methods, and how do they differ with respect to MEV? Which liquidation mechanism avoids MEV?
+- How sensitive are current debts to liquidation (by how much would an adversary need to manipulate a price oracle)? An MEV miner could try to manipulate an oracle to extract this MEV.
+- How can we design a possible liquidation mechanism that mitigates MEV?
```

Comment:
Possibly need to dive into the details of how you can avoid MEV in the first place. This thread may have useful info https://twitter.com/zmanian/status/1299460217919344640

Reply:
Would this be a case study that addresses reducing MEV?  Otherwise I think it may be overly specific for Flashbots-related-research, as it benefits one class of dapps over others (in this vein I also think this would be a great grant to seek funding from a liquidation-based-dapp if we are studying their protocol specifically and this could definitely be an avenue we explore; cc @obadiaa).

Reply:
One question that might be broad is more the 'negative' type of question:

Is it possible to create a securitization scheme that redistributes liquidation profits to an entire system's stakeholders without increasing the minimum risk held by any individual agent? If not, is the reason for this computational, mathematical, or financial?

Examples of why this might not work:
Computational --- finding such an equilibrium is coNP or EXPSPACE
Mathematical --- partitioning liquidation profits is impossible when you have discrete quantities (e.g. you can't find the correct Young Tableaux to represent redistribution)
Financial --- The expected social welfare under some pricing model is always negative